srv/components/system/storage/config.sls

```diff
@@ -18,10 +18,16 @@
 # Setup SWAP and /var/motr
 {% set node = grains['id'] %}
 
-# Make SWAP
-Ensure SWAP partition is unmounted:
-  cmd.run:
-    - name: swapoff -a && sleep 2
+# Disable system swap
+Deactivate and unmount system SWAP:
+  module.run:
+    # Deactivate SWAP
+    - mount.swapoff:
+      - name: /dev/dm-1
```

Comment:
Can we not use /dev/mapper/vg_sysvol-lv_swap instead /dev/dm-1?

Reply:
@83bhp 
SWAP unfortunately prefers `/dev/dm-1` for some reason for `swapoff`:
https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/srv/components/system/storage/config/config.sls

For `swapon` what you have suggested works and can be incorporated in teardown, if it helps.

Reply:
Oh, ok.